services/grafana-loki/0.48.5/defaults/cm.yaml

```diff
@@ -8,6 +8,8 @@ data:
   values.yaml: |
```

Comment:
a bit unrelated, but I realized that we probably have to bump the Loki k-apps version before 2.4 is released to avoid upgrading the workspace/project Loki when the management cluster is upgraded